### PR TITLE
Connect labels to form inputs properly

### PR DESCRIPTION
**Pull request by nickmccurdy**
This was originally part of [testing-jest](https://github.com/prettier/prettier-chrome-extension/tree/testing-jest) but it's still an important accessibility fix

### CODE REVIEW

File: extension/src/options/index.js [hunk 1]
```diff
@@ -58,95 +58,113 @@ function App() {
...
       </h1>
       <hr />
-      <label>Print width</label>
-      <input
-        type="number"
-        name="printWidth"
-        value={options.printWidth}
-        onChange={handleChange}
-      />
+      <label>
```
Comment (kaicataldo) [inline extension/src/options/index.js]: For my own learning, how is this better than using the `for` attribute?

Comment (nickmccurdy) [reply extension/src/options/index.js]: Moving inputs inside labels automatically associates them in some browsers and accessibility tools. However in theory it's better to do both, if you prefer.

Comment (kaicataldo) [reply extension/src/options/index.js]: As long as it works for Chrome, I think we're good :)

**Comment by kaicataldo** [comment]
LGTM, thanks!

**Pull request merged**